src/settings-ui/Microsoft.PowerToys.Settings.UI/Views/PowerLauncherPage.xaml

```diff
@@ -187,8 +200,7 @@
                        Visibility="{x:Bind ViewModel.ShowPluginsLoadingMessage, Mode=OneWay, Converter={StaticResource BoolToVisibilityConverter}}"
                        TextWrapping="Wrap"
                        Margin="{StaticResource SmallTopMargin}"/>
-
-            <ListView ItemsSource="{x:Bind Path=ViewModel.Plugins, Mode=OneWay}"                
+            <ListView ItemsSource="{x:Bind Path=ViewModel.Plugins, Mode=OneWay}"
```

Comment:
When we're searching, it's possible that the list will be cleared (no results) and the whole page resizes so that the search is at the bottom of the page and further results are not shown.
I think having some minumum height there might help this behavior.
Something like `MinHeight="512"` in the `ListView` would prevent the space for the list getting to small when search doesn't find anything.

What do you think?

Reply:
Nice catch! I have updated the PR 😃